test/toolchainstatus_assertions.go

```diff
@@ -60,6 +60,16 @@ func (a *ToolchainStatusAssertion) HasConditions(expected ...toolchainv1alpha1.C
 	return a
 }
 
+// HasMetric verifies that the `ToolchainStatus.Status.Metrics` has the given key/value (where the value is serialized if it is a map)
+func (a *ToolchainStatusAssertion) HasMetric(key string, value toolchainv1alpha1.Metric) *ToolchainStatusAssertion {
+	err := a.loadToolchainStatus()
+	require.NoError(a.t, err)
+	require.NotNil(a.t, a.toolchainStatus.Status)
+	require.NotNil(a.t, a.toolchainStatus.Status.Metrics)
+	assert.Equal(a.t, value, value)
```

Comment:
This will be always true :wink: 
I know that you used my [suggestion](https://github.com/codeready-toolchain/host-operator/pull/401#discussion_r615801565) - I didn't notice that in your original code - sorry :-)  
```suggestion
	assert.Equal(a.t, value, a.toolchainStatus.Status.Metrics[key])
```

Reply:
btw, is there any significant difference between this method and the `HasUsersPerActivations` one?

Reply:
fixed in f2ed54f

Reply:
no, indeed. I replaced `HasMetric` with `HasUsersPerActivations` everywhere in 5142600